docs/getting-started-apm-server.asciidoc

```diff
@@ -232,10 +232,36 @@ Any changes are automatically appended to the `apm-server.yml` configuration fil
 Full details are available in the {cloud}/ec-manage-apm-settings.html[APM user settings] documentation.
 
 [float]
-[[index-template-config]]
 ==== Self installation
 
-It is recommend that you run the `setup` command before starting {beatname_uc}.
+[float]
+===== APM integration for Elastic Agent
+
+If you're running APM Server as an Elastic Agent integration, you can configure settings via
+{fleet}/agent-policy.html[Elastic Agent policies].
+
+[float]
+===== Standalone
+
+You can edit the `apm-server.yml` configuration file to customize it to your needs.
+The location of this file varies by platform, but the <<directory-layout>> will help you locate it.
+All available configuration options are outlined in
+{apm-server-ref-v}/configuring-howto-apm-server.html[configuring APM Server].
+
+[float]
+====== Standalone setup (data streams)
+
+If you're running APM Server as a standalone process, we recommend that you configure <<data_streams.enabled>>
+to write events to data streams in the same way as the <<apm-integration>>. You will need to install the APM
+integration package to set up Elasticsearch before APM Server becomes operative.
+
+[float]
+[[index-template-config]]
+====== Standalone setup (classic indices)
+
+deprecated::[7.16.0,Users should now use the <<apm-integration>> or configure <<data_streams.enabled>>.]
+
+It is recommended that you run the `setup` command before starting {beatname_uc}.
```

Comment:
Can we change/remove this sentence here? It is confusing to say it is deprecated and it is recommended?

Reply:
I agree it's confusing. I'm hoping @bmorelli25 can work some magic later :smile: 

For now I've just prefixed this with "When running APM Server standalone with classic indices".